recipe/meta.yaml

```diff
@@ -20,15 +20,20 @@ requirements:
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - cython                                 # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
```

Comment:
Based on a quick look at SciPy

https://github.com/conda-forge/scipy-feedstock/blob/9401aa4ee5d9a22c2c9d219e5fb6dbd580f30207/recipe/meta.yaml#L56

You will probably need meson-python in the `build_platform!=target platform` area

Reply:
I think so too, I also move `trx-python`. it needs to be only on `run requirements` and not `host requirements`